roles/kubernetes/node/templates/manifests/kube-proxy.manifest.j2

```diff
@@ -42,7 +42,7 @@ spec:
     - --cluster-cidr={{ kube_pods_subnet }}
     - --proxy-mode={{ kube_proxy_mode }}
     - --oom-score-adj=-998
-    - --healthz-bind-address=127.0.0.1
+    - --healthz-bind-address={{ kube_proxy_healthz_bind_address }}
```

Comment:
+ @mattymo why are we binding to 127.0.0.1 ? the default configuration is 0.0.0.0: https://github.com/kubernetes/kubernetes/pull/44968#issue-117796942

Can we just remove this flag and leave the default?

Reply:
i think this should be configurable, let's say we don't want even health been accessible from outside of cluster.

Reply:
I'm indifferent here, but leaving it configurable is good